pkg/proof/README.md

```diff
@@ -0,0 +1,193 @@
+# Proofs
```

Comment:
we might want to name this something else, as imo this is a summary of commitments data and proofs of inclusion

Reply:
I called it proofs to be the same as the package name, I guess that's the convention for package readmes, no?